GitHubProjects.ps1

```diff
@@ -390,6 +386,13 @@ filter New-GitHubProject
         $hashBody.add('body', $Description)
     }
 
+    if (-not $PSCmdlet.ShouldProcess($ProjectName, 'Create GitHub Project'))
+    {
+        return
+    }
+
+    Write-InvocationLog
```

Comment:
And here we have two `Write-InvocationLog` calls...one here, and one still at the top of the function (line 346).

Reply:
First one removed.